### PR TITLE
US07_StepDefs added

### PR DESCRIPTION
**Pull request by SForogh**

### CODE REVIEW

File: src/test/java/com/library/steps/BooksStepDefs.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                "order by 2 desc";
+
+        DB_Util.runQuery(query);
+
+        String actualBookName = borrowedBookName;
+        String expectedBookName = DB_Util.getFirstRowFirstColumn();
+        Assert.assertTrue(expectedBookName.contains(actualBookName));
+        /*
+        List<String> expectedBorrowedBookList = DB_Util.getColumnDataAsList("Name");
+        Assert.assertTrue(expectedBorrowedBookList.contains(actualBookName));
```
Comment (latifaforozan) [inline src/test/java/com/library/steps/BooksStepDefs.java:65]: Do you need these list and assertion? for having clean code better to delete them.

**Review by latifaforozan — APPROVED** [review]
Great work Sohaila.

**Comment by Daniz2000** [comment]
I reviewed. Great work!

**Review by Daniz2000 — APPROVED** [review]

**Comment by hamidalakbarov** [comment]
Great work Sohaila, merging your branch.

File: src/test/java/com/library/steps/BooksStepDefs.java (hunk 1 above)
Comment (SForogh) [reply src/test/java/com/library/steps/BooksStepDefs.java:65]: I should have probably mentioned it as a 2nd solution.

**Pull request merged**